hazelcast/src/main/java/com/hazelcast/map/impl/MapService.java

```diff
@@ -199,9 +205,9 @@ public void rollbackTransaction(UUID transactionId) {
         transactionalService.rollbackTransaction(transactionId);
     }
 
-    @Override
```

Comment:
This method is still overriding the interface.

Reply:
Oh, thanks. Added back.